### DISCUSSION TITLE
s3authserver not working normally and gateway time-out

### DISCUSSION THREAD

**Discussion by h3czxp** [Seagate/cortx-s3server#1758] [Q&A]
1.When test the cortx cluster for about an hour using tool warp, the s3authserver is not working normally. 

2.When putting a large number of objects, the tool warp occurs error "gateway time-out", and the s3server occurs error "S3 Get request failed. HTTP status code = 404".

How to solve the above problems?

Thanks
Xianpeng Zhou

**Comment by hessio** [comment]
This is being resolved in this issue #1759
